plutus-metatheory/src/Builtin/Constant/AtomicType.lagda.md

```diff
@@ -37,3 +44,18 @@ data AtomicTyCon : Set where
 decAtomicTyCon : DecidableEquality AtomicTyCon
 unquoteDef decAtomicTyCon = defDec (quote AtomicTyCon) decAtomicTyCon
 ```
+
+## Meaning of Atomic Constants
+
+```
+⟦_⟧at : AtomicTyCon → Set
+⟦ aInteger ⟧at = ℤ
+⟦ aBytestring ⟧at = ByteString
+⟦ aString ⟧at = String
```

Comment:
I'm not sure it would change anything. Is there any difference aside from performance? The strings get converted to Text to use Haskell functions such as pretty-printing.

Reply:
> Is there any difference aside from performance? 

Accuracy? It's already confusing on the implementation side that `String` maps to `Text` and here we add a bunch more confusion on top.

It doesn't really matter, I'm just nitpicking.

Reply:
I'll make a ticket to fix this in the future.

Reply:
Done. PLT-6378.